scripts/globals/moblights.lua

```diff
@@ -0,0 +1,601 @@
+require("scripts/globals/abyssea")
+require("scripts/globals/status")
+require("scripts/globals/settings")
+
+tpz = tpz or {}
+tpz.abyssea_mob = tpz.abyssea_mob or {}
+
+-- if the amount is 100 or 50 it is a special number and is randomised as per retail
+tpz.abyssea_mob.info =
+{
+    ------------------------------------------------------------------------------------------------------------
+    -- Abyssea - Konschtat
+    ------------------------------------------------------------------------------------------------------------
+    ["Ab_xzomit"]              = {azure = 8,   pearl = 5,   ruby = 16,  amber = 8  }, -- Xzomits
```

Comment:
I would handled this information the way we handle mob 2hrs - input passed to a mixin, rather than trying to name every mob in every abyssea zone in a global table like this.